libs/components/pages/src/lib/modules/action-hub/types/page-modal-link.ts

```diff
@@ -0,0 +1,20 @@
+import { SkyModalConfigurationInterface } from '@skyux/modals';
+
+import { SkyPageLinkInterface } from './page-link-interface';
+
+/**
+ * Displays links to related information or recently accessed items.
+ */
+export interface SkyPageModalLink extends SkyPageLinkInterface {
+  /**
+   * Specifies the link text.
+   */
+  label: string;
```

Comment:
Since `label` is a part of `SkyPageLinkInterface` it is redundant to list it here too, correct?

Reply:
Good catch.